components/public-api/gitpod/v1/prebuild.proto

```diff
@@ -15,6 +15,9 @@ service PrebuildService {
   rpc GetPrebuild(GetPrebuildRequest) returns (GetPrebuildResponse) {}
   rpc ListPrebuilds(ListPrebuildsRequest) returns (ListPrebuildsResponse) {}
   rpc WatchPrebuild(WatchPrebuildRequest) returns (stream WatchPrebuildResponse) {}
+
+  // ListOrganizationPrebuilds lists all prebuilds of an organization
+  rpc ListOrganizationPrebuilds(ListOrganizationPrebuildsRequest) returns (ListOrganizationPrebuildsResponse) {}
```

Comment:
Ideally we would enhance the `ListPrebuilds` call with a scope argument that can be organization level or configuration level

Reply:
@svenefftinge I agree it would be unfortunate if we have to introduce another method when `ListPrebuilds` sounds like it should cover both. I don't have an issue with merging the props, although some like `workspace_id` and `git_ref` I would like to get rid of for our global prebuild use cases (at least for now).
Best would most likely be to redo the `ListPrebuilds` method entirely so that it supports proper pagination and supports filters. I am not sure if it would be fine to break this one, but I assume it's fine since nobody is using it externally, so refactoring our Projects code should be enough.

Originally, I separated this into its own method because I thought the effort on backporting the changes onto `ListPrebuilds` is too high, but think that your point about using just one method for all listings makes sense. 

If we do decide to go with only `ListPrebuilds`, I think a `configurationId` filter would fit in nicely as well (or are there any clear advantages of having a `scope` for this?). This would enable us to add a dropdown for the desired configuration to the global list in the future.